packages/docs/src/components/sidebar/navigationRoutes.ts

```diff
@@ -342,6 +342,10 @@ export const navigationRoutes: NavigationRoute[] = [
         name: 'dropdown',
         displayName: 'menu.dropdown',
```

Comment:
<img width="370" alt="image" src="https://user-images.githubusercontent.com/5394573/183222025-a013385b-b8b4-402a-8d58-a347770da37b.png">

Filter looks broken. I would assume a couple of things:
* `filter-by` prop should define how filtering would work. (could be string or function)
* default value should be the same as for `text-by` (as user would search by text displayed normally)